### DISCUSSION TITLE
Generating SEB config file outside SEB configuration tool.

### DISCUSSION THREAD

**Discussion by ghost** [SafeExamBrowser/seb-win-refactoring#670] [Q&A]
I encountered challenges while trying to generate SEB config files within the exam app that I am currently developing. I referred to the documentation available at https://safeexambrowser.org/developer/seb-file-format.html but faced certain difficulties. To proceed with the SEB config generation, I am in need of further insights and documentation. Specifically, I would greatly appreciate information on the required XML structure and the encryption process encryption requirements.

**Comment by dbuechel** [comment]
All required information should actually be specified on that page, in particular the XML structure is a property list (plist, see e.g. https://en.wikipedia.org/wiki/Property_list). In case there is some particular resp. specific question, please do feel free to post it below.

**Comment by jeroenhabets** [comment]
@abhi8170 also in case you understand C#, earlier @dbuechel provided us with some helpful pointers here: https://github.com/SafeExamBrowser/seb-win-refactoring/issues/591#issuecomment-1484919030 (though we have not pursuit this further ourselves also due to other prios)
